### ISSUE TITLE
Instructions for dracut/non debian-based distros

### ISSUE THREAD

**Issue by dgudim** [full-disclosure/android-luks#1]
Hey, this is a really cool project. I am looking to set it up on arch with dracut, however, was unable to do so. Are you planning to make instructions for non debian-based distros/for different initramfs generators?

**Comment by full-disclosure** [comment]
Yes, there are plans to support more distros. Just need to find some free time

**Comment by full-disclosure** [comment]
By the way, we noticed that Ubuntu 24 now ships dracut by default and initramfs tools are actually just wrappers for dracut. Everything still works out of the box
